static/js/protein/chart.ts

```diff
@@ -219,6 +240,76 @@ function addYLabel(
     .text(labelText);
 }
 
+export function drawTissueLegend(id: string, data: ProteinStrData[]): void {
+  // checks if the data is empty or not
+  if (_.isEmpty(data)) {
+    return;
+  }
+  const svg = d3
+    .select("#tissue-score-legend")
+    .append("svg")
+    .attr("width", GRAPH_WIDTH_XL)
+    .attr("height", GRAPH_HEIGHT_XS);
+  const data0 = d3.keys(TISSUE_LEGEND_DICT);
+  // slicing the dictionary in half to display the legend in two rows
```

Comment:
Yes, I checked that. The legend behaves the same as the graphs would. If I shrink or widen the window, both the graph and legend get cut off or stretched respectively!

Reply:
<img width="803" alt="Screen Shot 2022-06-27 at 1 46 25 PM" src="https://user-images.githubusercontent.com/57412795/176013783-0011d36e-8206-4f9d-bbbf-c640a415e100.png">
Here's how it looks like when shrinking the window. You're right that we shouldn't be experiencing this issue but it might be a good idea to fix it in another PR, with mobile screen view?

Reply:
You're right! Added as P0 to the protein to-do list